packages/kotti-ui/source/kotti-filters/components/FilterRow.vue

```diff
@@ -146,39 +163,25 @@ export default defineComponent<{
 			}
 		})
 		const valuePrefix = computed<KottiFieldNumber.Props['prefix']>(() =>
-			props.column?.type === KottiFilters.FilterType.CURRENCY ||
 			props.column?.type === KottiFilters.FilterType.FLOAT
 				? props.column.prefix
 				: null,
 		)
 		const valueStep = computed<KottiFieldNumber.Props['step']>(() => {
```

Comment:
logical question

was this only exposed from the filters API when we had the issue of having to add the `step` in order to allow for the field validation to work

if u remember internally we had a check `isStepMultiple` that was validating `value` for `KtFieldNumber` and before using `big` we had to pass `step` as a hack for it to work
if currently the problem is adding a value that has different decimal places and you want kotti to validate just that, then you need to expose decimalPlaces

and i wonder if it's now supposed to change this to expose `decimalPlaces` instead

`step` was there to logically allow the `+/-` buttons to add a certain step value

do u actually expose `+/-` or do you use `hideChangeProps` on `KtFieldNumber`

if you use `hideChangeButtons` for field of type `Float` then you don't need to expose `step`, you need to expose `decimalPlaces`
and if you say `decimalPlaces` is `3` then that's what we will support

Reply:
I checked the user-app

you don't use `hideChangeButtons` on number/float types


so my question is narrowed down to

do you need to limit `decimalPlaces` and thus validity of input on float numbers ?
By default it's `3`

I'm guessing internally for the number, you enforce `1` or you enforce the step / min / max to guarantee that's integer increments
> if you enforce the step, please enforce the `decimalPlaces` to `1` because the step by default is `1`

Reply:
Right, `step` was added due to the floating point precision issue with the `+/-` buttons. However, I'll keep it as `hideChangeButtons` is not being used.

Will expose `decimalPlaces` as well for `Float` types and `Integer` types will default to: `step = 1` and `decimalPlaces = 0`. 

Also, `prefix`, `suffix`, `decimalPlaces`, `step`, `currency` and `options` will default to `undefined` for types that don't support them.